source/precalculus/exercises/outcomes/EQ/EQ5/generator.sage

```diff
@@ -19,25 +19,24 @@ class Generator(BaseGenerator):
     #Square root property
     a=choice([2..5])
     h=choice([-7..-1,1..7])
-    c=choice([2..5])
+    c=choice([sqrt(i) for i in [2..12]])
```

Comment:
```suggestion
    c=choice([sqrt(i) for i in [2,3,5,6,7,8,10,11,12]])
```

Reply:
Maybe `c=choice([sqrt(i) for i in [2..12] if not i.is_square()])` is more readable?

Reply:
```suggestion
    c=choice([sqrt(i) for i in [2..12] if not i.is_square()])
```